bson/bsoncodec/time_codec_test.go

```diff
@@ -43,7 +43,13 @@ func TestTimeCodec(t *testing.T) {
 				actualTime := actual.Interface().(time.Time)
 				assert.Equal(t, actualTime.Location().String() == "UTC", tc.utc,
 					"Expected UTC: %v, got %v", tc.utc, actualTime.Location())
-				assert.Equal(t, now, actualTime, "expected time %v, got %v", now, actualTime)
+
+				if tc.utc {
```

Comment:
testify's `assert.Equal` found that `now` and `actualTime` varied in their location (time zone). We decode by default with a "UTC" time zone, but `time.Now()` return a "local" time zone. So, if `tc.utc` is true, we should be comparing to `now.UTC()`